account_invoice_pricelist/model/__init__.py

```diff
@@ -1,3 +1,6 @@
-# -*- encoding: utf-8 -*-
-
+# -*- coding: utf-8 -*-
+# © 2015-2016 GRAP <http://www.grap.coop>.
+# © 2017 Therp BV <http://therp.nl>.
+# License AGPL-3.0 or later <http://www.gnu.org/licenses/agpl.html>.
```

Comment:
Not blocking for me but here there is no content (just import), then authors and license can be removed (it's just my opinion ;-) )

Reply:
:+1: